src/ex2/ItemManager.js

```diff
@@ -0,0 +1,33 @@
+class ItemManager {
+    constructor() {
+        this.todoList = [];
+    }
+    
+    async addItem(todo){
+        if (!/^[0-9]+$/.test(todo))
```

Comment:
While this works, regex is hard to maintain and debug. I would use isNaN instead here